interact/axiom-fleet

```diff
@@ -1,34 +1,41 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+###################################################################
+# About	:
+#
+# axiom-fleet lets you spin up fleets of axiom instances in one or multiple regions.
+# You can specify the name of a fleet (fleet prefix) or have axiom choose for you.
+#
+# Examples: 
+#
+# axiom-fleet${Color_Off} # Spin up three instances, let axiom decide on the fleet prefix"
+# axiom-fleet javis -i 10${Color_Off} # Spin up 10 instances with a fleet prefix of javis, this will create 10 instances named javis01 to javis10."
+# axiom-fleet jerry -i 25 --regions nyc1,lon1,ams3,fra1${Color_Off} # Spin up 25 instances using round robbin region distribution"
+###################################################################
+
+###########################################################################################################
+# Header
+#
 AXIOM_PATH="$HOME/.axiom"
 source "$AXIOM_PATH/interact/includes/vars.sh"
 source "$AXIOM_PATH/interact/includes/functions.sh"
+source "$AXIOM_PATH/interact/includes/system-notification.sh"
+begin=$(date +%s)
+start="$(pwd)"
+BASEOS="$(uname)"
+account_path=$(ls -la $AXIOM_PATH/axiom.json | rev | cut -d " " -f 1 | rev)
+accounts=$(ls -l "$AXIOM_PATH/accounts/" | grep "json" | grep -v 'total ' | awk '{ print $9 }' | sed 's/\.json//g')
```

Comment:
what does this do?

Reply:
`start` is not being used atm ( might be better if we remove it to avoid any conflicts with [these](https://github.com/pry0cc/axiom/blob/master/interact/axiom-fleet#L161), however unlikely ).

`BASEOS` is used for Macs to tmp add `coreutils` [tools to path](https://github.com/pry0cc/axiom/blob/master/interact/axiom-fleet#L29-L34) to avoid using Mac OS X utilities, as there are slight differences between Mac OS X utilities and GNU core utilities. ( doesnt seem like we use any core utilities for this script but its helpful in case we want to )
`account_path` is useful for these [types of operations ](https://github.com/pry0cc/axiom/blob/master/interact/axiom-build#L65-L70) ( e.g. using JQ to edit the account.json in place) however we could just as easily edit `~/.axiom/axiom.json` since its just a symbolic link to the account.json anyways.

`accounts` just stores the available accounts and more than likely be removed too ( tbh not sure if this one is used in any other script )

Reply:
https://github.com/pry0cc/axiom/commit/ecbe51ba739ce55446027999f4cc0ff3f3bb2ec0